.github/workflows/dotnet-format.yml

```diff
@@ -0,0 +1,21 @@
+name: .NET format
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  dotnet-format:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 8.0.x
+    - name: Format
+      run: dotnet format src/Arcus.Testing.sln --verify-no-changes --verbosity diagnostic
```

Comment:
If I understand correctly, this `dotnet format --verify-no-changes` is just used to verify if the code adheres to the conventions listed in `.editorconfig`.   No changes are made; the build will just fail if conventions are broken.

Interesting, as we have something similar at a project, but we don't explicitly call `dotnet format` for this.